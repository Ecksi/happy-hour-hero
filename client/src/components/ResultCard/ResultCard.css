.resultCardContainer {
  margin: 60px;
  padding: 20px;
  display: flex;
  border-radius: 20px;
  justify-content: space-between;
  box-shadow: 1px 1px 1px 1px #eee;
  background: #fff; }

.resultCardContainer.selectedCard {
  border: 4px solid #fdb82a;
  box-shadow: none; }

.resultCardImage {
  width: 250px;
  height: auto;
  background: url(http://www.brothersbar.com/wp-content/uploads/2015/10/GALLERY-Stapleton.jpg);
  background-size: cover;
  border-radius: 5px; }

.resultCardInfo {
  margin-left: 20px;
  display: flex;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 20%; }

.resultCardInfo h2 {
  color: #666;
  margin: 0;
  font-size: 1.5rem;
  line-height: 2rem; }

.resultCardInfo .address {
  color: #b3b3b3;
  font-weight: 100;
  margin-top: 5px; }

.resultCardInfo h3 {
  color: #666;
  font-size: 1.1rem;
  margin: 20px 0 2px 0;
  font-weight: 400; }

.resultCardInfo .times {
  color: #b3b3b3;
  font-size: 1.1rem;
  margin-top: 0; }

.resultCardInfo .times span {
  color: #fdb82a; }

.fa-clock {
  font-size: 4rem;
  color: #fdb82a;
  margin-right: 10px; }

.resultCardClock {
  float: right;
  display: flex;
  height: 70px;
  align-items: center; }

.resultCardClock p {
  color: #b3b3b3;
  font-size: 1.2rem;
  margin: 5px; }

.resultCardStartTime {
  color: #666 !important;
  font-size: 1.2rem !important; }

.happyHourSpecialsContainer {
  width: 35%; }

.happyHourSpecials {
  height: 50px;
  background: #fdb82a;
  display: flex;
  padding: 5px 25px;
  text-align: center;
  margin-top: -20px;
  display: flex;
  align-items: center;
  justify-content: space-between;
  color: #fff;
  width: 77%;
  margin-left: 10px;
  font-size: 1rem;
  border-radius: 0 0 5px 5px; }

.happyHourSpecials span {
  font-size: 1.6rem;
  color: #e09b1f; }

.resultCardRightInfo {
  width: 19%;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: flex-end;
  text-align: right; }

.moreInfoButton {
  color: #666;
  text-decoration: none;
  font-size: 1.3rem;
  width: 100px;
  display: flex;
  padding: 5px 20px;
  border: 1px solid #b3b3b3;
  border-radius: 5px;
  justify-content: center; }

.miles {
  color: #999;
  font-style: italic;
  font-size: 0.9rem;
  margin: -10px 0 10px 0; }

@media only screen and (max-width: 1000px) and (min-width: 700px) {
  .resultCardRightInfo {
    width: 45%;
    flex-direction: column;
    justify-content: space-between;
    margin-top: 70px;
    margin-left: -90px; }
  .happyHourSpecials {
    width: 157%; }
  .resultCardInfo {
    width: 35%; }
  .moreInfoButton {
    margin-top: 20px; } }

@media only screen and (max-width: 700px) {
  .resultCardContainer {
    flex-direction: column;
    text-align: center;
    position: relative;
    padding: 20px 20px 90px 20px;
    margin: 30px; }
  .resultCardImage {
    width: 100%; }
  .resultCardInfo {
    margin-left: 0;
    width: 100%;
    margin-top: 30px; }
  .happyHourSpecialsContainer {
    width: 100%; }
  .happyHourSpecials {
    height: 50px;
    margin-left: 0;
    padding: 5px 25px;
    margin-top: 0px;
    color: #fff;
    width: 74%;
    display: flex;
    bottom: 25px;
    position: absolute;
    border-radius: 20px;
    -ms-flex-pack: distribute;
    justify-content: space-around; }
  .resultCardRightInfo {
    width: 100%;
    align-items: normal;
    flex-direction: column;
    align-items: center;
    padding-bottom: 25px;
    justify-content: space-between; }
  .moreInfoButton {
    height: 25px;
    font-size: 1.1rem; }
  .resultCardStartTime {
    font-size: 1rem !important; }
  .resultCardInfo h3 {
    margin: 10px 0 2px 0; }
  .fa-clock {
    font-size: 2.6rem;
    margin-left: 20px; }
  .resultCardClock {
    width: 100%;
    display: flex;
    margin-bottom: 50px;
    text-align: center;
    flex-direction: column; } }
