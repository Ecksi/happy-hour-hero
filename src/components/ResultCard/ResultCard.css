.resultCardContainer {
  margin: 60px;
  padding-bottom: 20px;
  border-bottom: 1px solid #ccc;
  display: flex;
  justify-content: space-between; }

.resultCardImage {
  width: 300px;
  height: 150px;
  background: url(http://www.brothersbar.com/wp-content/uploads/2015/10/GALLERY-Stapleton.jpg);
  background-size: cover;
  border-radius: 5px; }

.resultCardInfo {
  margin-left: 40px;
  display: flex;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 20%; }

.resultCardInfo h2 {
  color: #666;
  margin: 0;
  font-size: 2rem;
  line-height: 2rem; }

.resultCardInfo .address {
  color: #b3b3b3;
  font-weight: 100;
  margin-top: 10px; }

.resultCardInfo h3 {
  color: #666;
  font-size: 1.5rem;
  margin: 20px 0 2px 0;
  font-weight: 400; }

.resultCardInfo .times {
  color: #b3b3b3;
  font-size: 1.1rem;
  margin-top: 0; }

.resultCardInfo .times span {
  color: #fdb82a; }

.fa-clock {
  font-size: 4rem;
  color: #fdb82a;
  margin-right: 10px; }

.resultCardClock {
  float: right;
  display: flex;
  height: 70px;
  width: 20%;
  align-items: center; }

.resultCardClock p {
  color: #b3b3b3;
  font-size: 1.2rem;
  margin: 5px; }

.resultCardStartTime {
  color: #666 !important;
  font-size: 1.3rem !important; }

.happyHourSpecials {
  height: 50px;
  background: #fdb82a;
  padding: 5px 25px;
  border-radius: 0 0 5px 5px;
  display: flex;
  align-items: center;
  color: #fff;
  width: 25%;
  font-size: 1.1rem; }
