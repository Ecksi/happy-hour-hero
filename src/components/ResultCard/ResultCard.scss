.resultCardContainer {
  margin: 60px;
  padding-bottom: 20px;
  border-bottom: 1px solid #ccc;
  display: flex;
}

.resultCardImage {
  width: 300px;
  height: 150px;
  background: url(http://www.brothersbar.com/wp-content/uploads/2015/10/GALLERY-Stapleton.jpg);
  background-size: cover;
  border-radius: 5px;
}

.resultCardInfo {
  margin-left: 60px;
  display: flex;
  flex-direction: column;
}

.resultCardInfo h2 {
  color: #666;
  margin: 0;
  font-size: 2rem;
  line-height: 2rem;
}

.resultCardInfo .address {
  color: #b3b3b3;
  font-weight: 100;
  margin-top: 10px;
}

.resultCardInfo h3 {
  color: #666;
  font-size: 1.5rem;
  margin-top: 5px;
  font-weight: 400;
}



