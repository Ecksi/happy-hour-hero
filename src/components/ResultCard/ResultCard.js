import React, { Component } from 'react';
import { connect } from 'react-redux';
import './ResultCard.css';

class ResultCard extends Component {
  constructor (props) {
    super(props);
    
  }

  render() {
    const { restaurantName, address, image } = this.props;
    const backgroundImage = {backgroundImage: "url(" + image + ")"};

    return (
      <article className="resultCardContainer">
        <div className="resultCardImage" style={backgroundImage}>
        </div>
        <div className="resultCardInfo">
          <h2>{ restaurantName }</h2>
          <p className="address">{ address }</p>
          <h3>hours</h3>
          
        </div>
      </article>
    );
  }
}

export const mapStateToProps = (state) => ({
  filteredRestaurants: state.filteredRestaurants
});

export default connect(mapStateToProps)(ResultCard);
