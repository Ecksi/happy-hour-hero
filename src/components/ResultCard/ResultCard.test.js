import React from 'react';
import { shallow } from 'enzyme';
import { ResultCard } from './ResultCard';

jest.mock('moment', () => () => ({format: () => '1700'}));

describe('ResultCard', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    mockProps = {
      startTime: '1200',
      endTime: '1800'
    };
    
    wrapper = shallow(<ResultCard {...mockProps}/>);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe('secondsToTime', () => {
    it('should call this.countdown every second', () => {
      const result = wrapper.instance().secondsToTime(30000);
      const expected = {
        hours: 8,
        minutes: 20,
        seconds: 0
      };

      expect(result).toEqual(expected);
    });
  });

  describe('setTimeUntilRemaining', () => {
    it('should set state of currentlyHappyHour to true when happy hour is going on', () => {
      
      wrapper.instance().setTimeUntilRemaining();

      expect(wrapper.state('currentlyHappyHour')).toEqual(true);
    });

    it('should set state of currentlyHappyHour to false when happy hour is not going on', () => {
      mockProps.endTime = '1500';
      
      wrapper = shallow(<ResultCard {...mockProps}/>);

      wrapper.instance().setTimeUntilRemaining();

      expect(wrapper.state('currentlyHappyHour')).toEqual(false);
    });
  });
});