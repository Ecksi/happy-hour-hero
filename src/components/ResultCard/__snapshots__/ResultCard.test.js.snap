// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultCard matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ResultCard
    endTime="1800"
    startTime="1200"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="resultCardImage"
          style={
                    Object {
                              "backgroundImage": "url(undefined)",
                            }
          }
/>,
        <div
          className="resultCardInfo"
>
          <h2 />
          <p
                    className="address"
          />
          <p
                    className="miles"
          >
                     Miles
          </p>
          <h3>
                    happy hour times
          </h3>
          <p
                    className="times"
          >
                    mon-fri 
                    <span />
          </p>
</div>,
        <div
          className="happyHourSpecialsContainer"
>
          <div
                    className="happyHourSpecials"
          >
                    <p
                              className="drinkSpecial"
                    >
                              No Drink Specials
                    </p>
                    <span>
                              |
                    </span>
                    <p
                              className="foodSpecial"
                    >
                              No Food Specials
                    </p>
          </div>
</div>,
        <div
          className="resultCardRightInfo"
>
          <div
                    className="resultCardClock"
          >
                    <i
                              className="far fa-clock"
                    />
                    <span>
                              <p>
                                        Ends in:
                                         
                              </p>
                              <p
                                        className="resultCardStartTime"
                              >
                                        0
                                        hrs 
                                        0
                                        mins 
                                        0
                                        secs
                              </p>
                    </span>
          </div>
          <a
                    className="moreInfoButton"
                    href="#"
                    onClick={[Function]}
          >
                    More Info
          </a>
</div>,
      ],
      "className": "resultCardContainer resultCardundefined",
      "id": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "resultCardImage",
          "style": Object {
            "backgroundImage": "url(undefined)",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2 />,
            <p
              className="address"
/>,
            <p
              className="miles"
>
               Miles
</p>,
            <h3>
              happy hour times
</h3>,
            <p
              className="times"
>
              mon-fri 
              <span />
</p>,
          ],
          "className": "resultCardInfo",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "address",
            },
            "ref": null,
            "rendered": null,
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                undefined,
                " Miles",
              ],
              "className": "miles",
            },
            "ref": null,
            "rendered": Array [
              undefined,
              " Miles",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "happy hour times",
            },
            "ref": null,
            "rendered": "happy hour times",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "mon-fri ",
                <span />,
              ],
              "className": "times",
            },
            "ref": null,
            "rendered": Array [
              "mon-fri ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "span",
              },
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="happyHourSpecials"
>
            <p
                        className="drinkSpecial"
            >
                        No Drink Specials
            </p>
            <span>
                        |
            </span>
            <p
                        className="foodSpecial"
            >
                        No Food Specials
            </p>
</div>,
          "className": "happyHourSpecialsContainer",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="drinkSpecial"
>
                No Drink Specials
</p>,
              <span>
                |
</span>,
              <p
                className="foodSpecial"
>
                No Food Specials
</p>,
            ],
            "className": "happyHourSpecials",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "No Drink Specials",
                "className": "drinkSpecial",
              },
              "ref": null,
              "rendered": "No Drink Specials",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "|",
              },
              "ref": null,
              "rendered": "|",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "No Food Specials",
                "className": "foodSpecial",
              },
              "ref": null,
              "rendered": "No Food Specials",
              "type": "p",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="resultCardClock"
>
              <i
                            className="far fa-clock"
              />
              <span>
                            <p>
                                          Ends in:
                                           
                            </p>
                            <p
                                          className="resultCardStartTime"
                            >
                                          0
                                          hrs 
                                          0
                                          mins 
                                          0
                                          secs
                            </p>
              </span>
</div>,
            <a
              className="moreInfoButton"
              href="#"
              onClick={[Function]}
>
              More Info
</a>,
          ],
          "className": "resultCardRightInfo",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <i
                  className="far fa-clock"
/>,
                <span>
                  <p>
                                    Ends in:
                                     
                  </p>
                  <p
                                    className="resultCardStartTime"
                  >
                                    0
                                    hrs 
                                    0
                                    mins 
                                    0
                                    secs
                  </p>
</span>,
              ],
              "className": "resultCardClock",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "far fa-clock",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Ends in:
                       
</p>,
                    <p
                      className="resultCardStartTime"
>
                      0
                      hrs 
                      0
                      mins 
                      0
                      secs
</p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Ends in:",
                        " ",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      "Ends in:",
                      " ",
                    ],
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        0,
                        "hrs ",
                        0,
                        "mins ",
                        0,
                        "secs",
                      ],
                      "className": "resultCardStartTime",
                    },
                    "ref": null,
                    "rendered": Array [
                      0,
                      "hrs ",
                      0,
                      "mins ",
                      0,
                      "secs",
                    ],
                    "type": "p",
                  },
                ],
                "type": "span",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "More Info",
              "className": "moreInfoButton",
              "href": "#",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "More Info",
            "type": "a",
          },
        ],
        "type": "div",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="resultCardImage"
            style={
                        Object {
                                    "backgroundImage": "url(undefined)",
                                  }
            }
/>,
          <div
            className="resultCardInfo"
>
            <h2 />
            <p
                        className="address"
            />
            <p
                        className="miles"
            >
                         Miles
            </p>
            <h3>
                        happy hour times
            </h3>
            <p
                        className="times"
            >
                        mon-fri 
                        <span />
            </p>
</div>,
          <div
            className="happyHourSpecialsContainer"
>
            <div
                        className="happyHourSpecials"
            >
                        <p
                                    className="drinkSpecial"
                        >
                                    No Drink Specials
                        </p>
                        <span>
                                    |
                        </span>
                        <p
                                    className="foodSpecial"
                        >
                                    No Food Specials
                        </p>
            </div>
</div>,
          <div
            className="resultCardRightInfo"
>
            <div
                        className="resultCardClock"
            >
                        <i
                                    className="far fa-clock"
                        />
                        <span>
                                    <p>
                                                Ends in:
                                                 
                                    </p>
                                    <p
                                                className="resultCardStartTime"
                                    >
                                                0
                                                hrs 
                                                0
                                                mins 
                                                0
                                                secs
                                    </p>
                        </span>
            </div>
            <a
                        className="moreInfoButton"
                        href="#"
                        onClick={[Function]}
            >
                        More Info
            </a>
</div>,
        ],
        "className": "resultCardContainer resultCardundefined",
        "id": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "resultCardImage",
            "style": Object {
              "backgroundImage": "url(undefined)",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2 />,
              <p
                className="address"
/>,
              <p
                className="miles"
>
                 Miles
</p>,
              <h3>
                happy hour times
</h3>,
              <p
                className="times"
>
                mon-fri 
                <span />
</p>,
            ],
            "className": "resultCardInfo",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "address",
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  undefined,
                  " Miles",
                ],
                "className": "miles",
              },
              "ref": null,
              "rendered": Array [
                undefined,
                " Miles",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "happy hour times",
              },
              "ref": null,
              "rendered": "happy hour times",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "mon-fri ",
                  <span />,
                ],
                "className": "times",
              },
              "ref": null,
              "rendered": Array [
                "mon-fri ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "span",
                },
              ],
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="happyHourSpecials"
>
              <p
                            className="drinkSpecial"
              >
                            No Drink Specials
              </p>
              <span>
                            |
              </span>
              <p
                            className="foodSpecial"
              >
                            No Food Specials
              </p>
</div>,
            "className": "happyHourSpecialsContainer",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p
                  className="drinkSpecial"
>
                  No Drink Specials
</p>,
                <span>
                  |
</span>,
                <p
                  className="foodSpecial"
>
                  No Food Specials
</p>,
              ],
              "className": "happyHourSpecials",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "No Drink Specials",
                  "className": "drinkSpecial",
                },
                "ref": null,
                "rendered": "No Drink Specials",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "|",
                },
                "ref": null,
                "rendered": "|",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "No Food Specials",
                  "className": "foodSpecial",
                },
                "ref": null,
                "rendered": "No Food Specials",
                "type": "p",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="resultCardClock"
>
                <i
                                className="far fa-clock"
                />
                <span>
                                <p>
                                                Ends in:
                                                 
                                </p>
                                <p
                                                className="resultCardStartTime"
                                >
                                                0
                                                hrs 
                                                0
                                                mins 
                                                0
                                                secs
                                </p>
                </span>
</div>,
              <a
                className="moreInfoButton"
                href="#"
                onClick={[Function]}
>
                More Info
</a>,
            ],
            "className": "resultCardRightInfo",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="far fa-clock"
/>,
                  <span>
                    <p>
                                        Ends in:
                                         
                    </p>
                    <p
                                        className="resultCardStartTime"
                    >
                                        0
                                        hrs 
                                        0
                                        mins 
                                        0
                                        secs
                    </p>
</span>,
                ],
                "className": "resultCardClock",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-clock",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Ends in:
                         
</p>,
                      <p
                        className="resultCardStartTime"
>
                        0
                        hrs 
                        0
                        mins 
                        0
                        secs
</p>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          "Ends in:",
                          " ",
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        "Ends in:",
                        " ",
                      ],
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          0,
                          "hrs ",
                          0,
                          "mins ",
                          0,
                          "secs",
                        ],
                        "className": "resultCardStartTime",
                      },
                      "ref": null,
                      "rendered": Array [
                        0,
                        "hrs ",
                        0,
                        "mins ",
                        0,
                        "secs",
                      ],
                      "type": "p",
                    },
                  ],
                  "type": "span",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "More Info",
                "className": "moreInfoButton",
                "href": "#",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "More Info",
              "type": "a",
            },
          ],
          "type": "div",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
